src/transform/transform.ts

```diff
@@ -600,6 +605,9 @@ function createResponseType(codeModel: CodeModel, group: OperationGroup, op: Ope
       if (isSchemaResponse(response) && response.schema.language.go!.responseType.value) {
         name = generateLROPollerName(response);
       }
+      if (isPageableOperation(op)) {
```

Comment:
Put in `generateLROPollerName()` to keep naming in one place.

Reply:
Done